doc/builders/packages/firefox.section.md

```diff
@@ -2,11 +2,12 @@
 
 ## Build wrapped Firefox with extensions and policies {#build-wrapped-firefox-with-extensions-and-policies}
 
-The `wrapFirefox` function allows to pass policies, preferences and extension that are available to firefox. With the help of `fetchFirefoxAddon` this allows build a firefox version that already comes with addons pre-installed:
+The `wrapFirefox` function allows to pass policies, preferences and extension that are available to Firefox. With the help of `fetchFirefoxAddon` this allows build a Firefox version that already comes with addons pre-installed:
 
 ```nix
 {
-  myFirefox = wrapFirefox firefox-unwrapped {
+  # Nix firefox addons only work in ESR build.
```

Comment:
Only in ESR builds meaning 91.0 ESR works, but 91.0 does not?

Reply:
correct

Reply:
Maybe say 78/91 ESR to be clearer here. We can likely expect 91 ESR to be the last ESR series to allow for unsigned addons.

Reply:
Hmm I don't think so because there needs to be a way for corporations to deploy internal addons not published in the firefox addon store~ I rewrote it to `Nix firefox addons only work in the firefox-esr package` . Is that okay?